### PR TITLE
feat(pool): show all pools without web3 wallet

### LINKED ISSUE #86: Showing all pools even without Metamask installed
#### What would you like to be added:

In the `Pool` page, we need to show all pools even if user doesn't have Metamask installed or on the wrong network. It's ok even if it shows that the user is having zero-value liquidity positions.

### PR DESCRIPTION
**Pull request by gjj**
#### What this PR does / why we need it:

In this PR, we're showing the three LP pools without requiring the user's Web3 wallet. It uses Infura internally, and is already configured by Uniswap.

image

#### Which issue(s) does this PR fixes?:

Fixes #86

### CODE REVIEW

**Review by chee-chyuan — DISMISSED** [review]

**Comment by chee-chyuan** [comment]
eh the pool details isnt showing for me when wallet is not connected

**Comment by gjj** [comment]
I realise that too after deploying. Works on localhost but not on preview, might be because of API key. Will investigate further, thanks!

**Comment by gjj** [comment]
Found out I updated the wrong API key. It should be updated in `.env.production`.

API key belongs to my corp account. Have already setup domain whitelisting on my end to allow:

- liquidity.dfi.dev
- uniswap.defichain.com
- localhost:3000
- *--liquidity-dfi.netlify.app

**Review by chee-chyuan — APPROVED** [review]

**Comment by chee-chyuan** [comment]
> eh the pool details isnt showing for me when wallet is not connected

@gjj  was infura api key the issue?

**Comment by gjj** [comment]
@chee-chyuan indeed, thanks for pointing out earlier! `.env` and `.env.production` had two separate config for API keys. I updated `.env` but didn't check `.env.production` was still using Uniswap's key.

Have since updated `.env.production` to use our own Infura key with whitelisted origins.

File: .env.production [hunk 1]
```diff
@@ -1,5 +1,5 @@
 REACT_APP_CHAIN_ID="1"
-REACT_APP_NETWORK_URL="https://mainnet.infura.io/v3/099fc58e0de9451d80b18d7c74caa7c1"
+REACT_APP_NETWORK_URL="https://mainnet.infura.io/v3/1a409f1239b1440aa0843de4078a9ff4"
```
Comment (gjj) [inline .env.production:2]: @chee-chyuan

**Pull request merged**